weaver.go

```diff
@@ -85,10 +110,8 @@ func init() {
 		ServerStubFn: func(any, func(uint64, float64)) codegen.Server { return nil },
 	})
 
-	// Add a trivial /healthz handler to the default mux.
-	http.HandleFunc("/healthz", func(w http.ResponseWriter, _ *http.Request) {
-		w.Write([]byte("ok")) //nolint:errcheck // response write error
-	})
+	// Register HealthzHandler in the default ServerMux.
+	http.HandleFunc(HealthzURL, HealthzHandler)
```

Comment:
I think we should move this into weaver.Init so that no changes are made if the user just happened to link in this package.

But we will some protection to make sure we only do it once (at least with weavertest, weaver.Init can be called more than once.)

Reply:
Good point, done.